### DISCUSSION TITLE
Printing Error in the Assimp-docs 5.0.1 PDF : Assimp

### DISCUSSION THREAD

**Discussion by yeokaiwei** [assimp/assimp#3676] [General]
![Capture](https://user-images.githubusercontent.com/36597018/109172975-4b8dae00-77be-11eb-8bc3-053e90d6cc9f.PNG)

Hi,
There's a printing error with the Assimp PDF.

**Comment by kimkulling** [comment]
Oh, good to know that. I will fix that. Thanks!

**Comment by kimkulling** [comment]
Ok, thanks! I will take care.
